src/status_im/components/common/styles.cljs

```diff
@@ -77,12 +72,33 @@
           :opacity       0.6
           :padding-left  8
           :padding-right 5
-          :color         text4-color}))
+          :color         styles/text4-color}))
 
 (defstyle form-spacer
   {:ios     {:height 16}
    :android {:height 11}})
 
 (defstyle list-header-footer-spacing
-  {:android {:background-color color-white
+  {:android {:background-color styles/color-white
              :height           8}})
+
+(def network-container
+  {:flex-direction     :row
+   :padding-horizontal 13
+   :padding-vertical   11
+   :align-items        :center})
+
+(defn network-text [text-color]
+  {:flex           1
+   :color          (or text-color styles/color-black)
+   :letter-spacing -0.2
+   :font-size      14
+   :margin-left    16})
+
+(def network-icon
+  {:width            40
+   :height           40
+   :border-radius    (/ 40 2)
```

Comment:
why not just `20`? if this value depends on previous we could define a constant or add `let` form here